dotcom-rendering/src/components/StickyLiveblogAsk.importable.tsx

```diff
@@ -0,0 +1,273 @@
+/**
+ * This is a hard coded support ask as a test to see if it works.
+ * If it does, it may become a more standard feature.
+ */
```

Comment:
The official JSDoc tags are [`file` or `overview`](https://jsdoc.app/tags-file)

```suggestion
/**
 * @file
 * This is a hard coded support ask as a test to see if it works.
 * If it does, it may become a more standard feature.
 */
```

Reply:
Thanks, applied to branch skb/sticky-live-blog-support-asks